README.md

```diff
@@ -20,8 +20,18 @@
 [![Contributors](https://img.shields.io/github/contributors/hakancelikdev/pydbm)](https://github.com/hakancelikdev/pydbm/graphs/contributors)
 [![Last Commit](https://img.shields.io/github/last-commit/hakancelikdev/pydbm.svg)](https://github.com/hakancelikdev/pydbm/commits/main)
 
------
+----
 
 It provides a fast, simple, and convenient facility for your small-scale Python projects that need a database.
 
 For more information see: https://pydbm.hakancelik.dev/
+
+----
+Pydbm is a database management system that uses the dbm standard library from Python to provide interfaces to Unix databases in a pythonic way.
+It is designed for small-scale projects and is a light database, meaning it is not as feature-rich or powerful as other types of databases, such as relational databases.
+
+Pydbm is particularly useful for applications that need to store and retrieve simple data structures quickly,
+and is well-suited for developers working on small-scale projects that do not require the full functionality of a more complex database management system.
+
+Pydbm also includes an object-relational mapper (ORM), which allows developers to work with their database using objects and classes rather than raw commands.
```

Comment:
```suggestion
Pydbm is also an object-relational mapper (ORM), which allows developers to work with their database using objects and classes rather than raw commands.
```